src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -227,6 +227,9 @@
     <value>The {0}'s timestamp certificate has an unsupported signature algorithm.</value>
     <comment>0 - Signature friendly name</comment>
   </data>
+  <data name="VerifyError_TimestampUnsupportedDigestAlgorithm" xml:space="preserve">
+    <value>The timestamp signature has an unsupported digest algorithm.</value>
```

Comment:
Follow the example of the above resource string (`VerifyError_TimestampUnsupportedSignatureAlgorithm`):

`The {0}'s timestamp signature has an unsupported digest algorithm.`

That way, the full error message will be one of the following:

* The author primary signature's timestamp signature has an unsupported digest algorithm.
* The repository primary signature's timestamp signature has an unsupported digest algorithm.
* The repository countersignature's timestamp signature has an unsupported digest algorithm.

Reply:
Thanks for your explanation! I'll fix it.